.pre-commit-config.yaml

```diff
@@ -20,17 +45,8 @@ repos:
       args: ['operator-lint']
       pass_filenames: false
 
-- repo: https://github.com/dnephin/pre-commit-golang
-  rev: v0.5.1
-  hooks:
-    - id: go-fmt
-      exclude: ^vendor
-    - id: go-vet
-    - id: go-mod-tidy
-    - id: go-lint
```

Comment:
We are removing this but not adding `golint` back.

Reply:
Added back.